packages/@react-native-windows/cli/src/index.ts

```diff
@@ -6,6 +6,48 @@
 
 import * as fs from 'fs';
 import * as path from 'path';
+import * as findUp from 'find-up';
+import {randomBytes} from 'crypto';
+import * as appInsights from 'applicationinsights';
+
+appInsights.setup('795006ca-cf54-40ee-8bc6-03deb91401c3');
+const telClient = appInsights.defaultClient;
+if (!telClient.commonProperties.sessionId) {
+  telClient.commonProperties.sessionId = randomBytes(16).toString('hex');
+  function sanitizeStackTrace(envelope: any /*context: any*/): boolean {
```

Comment:
Not sure if app insights does this already, but user name is typically on the path of our projects. I don't know if the call stacks here have file names. We might have to sanitize these paths to exclude the username or computer name to avoid collecting PII.

Reply:
yep that's the point of the sanitizeStackTrace, remove most of the paths in stacks and leave only the leaf directory and filename since those won't have any PII